Tasks/Gradle/CodeAnalysis/Common/AnalysisResult.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Data class that describes code analysis results
```

Comment:
Note that all the files in Common can be used by both Maven and Gradle. Currently I've enabled the Gradle experience. I will refactor Maven to use these objects separately.